galgebra/mv.py

```diff
@@ -43,68 +43,6 @@
 })
 
 
-class _FmtResult:
```

Comment:
@utensil you were right, I should have put this in `printer.py` all along

Reply:
:)